src/solver/simulation/include/antares/solver/simulation/solver.hxx

```diff
@@ -339,6 +339,11 @@ void ISimulation<ImplementationType>::run()
         // for a single simulation
         study.resizeAllTimeseriesNumbers(1 + study.runtime->rangeLimits.year[Data::rangeEnd]);
         // Now, we will prepare the time-series numbers
+        if (not TimeSeriesNumbers::CheckNumberOfColumns(study))
```

Comment:
Good : resizing due to TS generation was already made at this stage

Reply:
Yes !